tools/generate_version.bat

```diff
@@ -1,7 +1,12 @@
-cd "%DUB_PACKAGE_DIR%"
+@echo off
+
+pushd %CD%
+cd /D "%DUB_PACKAGE_DIR%"

```

Comment:
Why is both `pushd` and `cd` necessary?

Reply:
I actually didn't know `pushd` also changes the directory, I only thought it would store the given path to a stack. So in this case, `pushd "%DUB_PACKAGE_DIR%"` should be enough.